src/GUI/UI/UIQuickReference.cpp

```diff
@@ -45,135 +42,103 @@ void GUIWindow_QuickReference::Update() {
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, pTextColor, fmt::format("{}", player.GetActualAC()), 84, 0);
+        pY += pFontHeight;
+
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), fmt::format("{:+}", player.GetActualAttack(false)), 84, 0);
+        pY += pFontHeight;
+
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), player.GetMeleeDamageString(), 84, 0);
+        pY += pFontHeight;
+
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), fmt::format("{:+}", player.GetRangedAttack()), 84, 0);
+        pY += pFontHeight;
+
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), player.GetRangedDamageString(), 84, 0);
+        pY += pFontHeight;
+
+        int pSkillsCount = 0;
         for (PLAYER_SKILL_TYPE j : AllSkills()) {
-            if (player->pActiveSkills[j])
+            if (player.pActiveSkills[j]) {
                 ++pSkillsCount;
+            }
         }
-        pTempString = fmt::format("{}", pSkillsCount);
-        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), pTempString, 84, 0);
-        pY = pFontHeight + pY;
-
-        if (!i)
-            pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {22, pY}, Color(), localization->GetString(LSTR_POINTS), 60, 0);
-        pTempString = fmt::format("{}", player->uSkillPoints);
-        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY},
-            player->uSkillPoints ? ui_character_bonus_text_color : ui_character_default_text_color, pTempString, 84, 0);
-        pY = pFontHeight + pY;
-
-        if (!i)
-            pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {22, pY}, Color(), localization->GetString(LSTR_COND), 60, 0);
-        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, pTextColor, localization->GetCharacterConditionName(player->GetMajorConditionIdx()), 84, 0);
-        pY = pFontHeight + pY;
-
-        if (!i)
-            pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {22, pY}, Color(), localization->GetString(LSTR_QSPELL), 60, 0);
-        if (player->uQuickSpell != SPELL_NONE)
-            pText = pSpellStats->pInfos[player->uQuickSpell].pShortName;
-        else
-            pText = localization->GetString(LSTR_NONE);
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, Color(), fmt::format("{}", pSkillsCount), 84, 0);
+        pY += pFontHeight;
+
+        pTextColor = player.uSkillPoints ? ui_character_bonus_text_color : ui_character_default_text_color;
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, pTextColor, fmt::format("{}", player.uSkillPoints), 84, 0);
+        pY += pFontHeight;
+
+        pGUIWindow_CurrentMenu->DrawTextInRect(pFontArrus, {pX, pY}, pTextColor, localization->GetCharacterConditionName(player.GetMajorConditionIdx()), 84, 0);
```

Comment:
Need to adjust pTextColor here

Reply:
Or rather, not change it in the previous code block

Reply:
We need to get condition color to draw here.